doc/schema/1.2/post_test_group.json

```diff
@@ -0,0 +1,92 @@
+{
+    "$schema": "http://api.kernelci.org/json-schema/1.2/post_test_group.json",
+    "id": "http://api.kernelci.org/json-schema/1.2/post_test_group.json",
+    "title": "test_group",
+    "description": "A test group JSON object",
+    "type": "object",
+    "properties": {
+        "version": {
+            "type": "string",
+            "description": "The version number of this JSON schema",
+            "enum": ["1.0", "1.2"]
+        },
+        "name": {
+            "type": "string",
+            "description": "The name given to this test group"
+        },
+        "lab_name": {
+            "type": "string",
+            "description": "The name of the lab executing this test group"
+        },
+        "time": {
+            "type": "number",
+            "description": "The number of seconds it took to execute the entire test group",
+            "default": -1
+        },
+        "job": {
+            "type": "string",
+            "description": "The name of the job (aka the git tree)"
+        },
+        "kernel": {
+            "type": "string",
+            "description": "The name of the kernel or the git describe value"
+        },
+        "defconfig": {
+            "type": "string",
+            "description": "The name of the defconfig"
+        },
+        "defconfig_full": {
+            "type": "string",
+            "description": "The full name of the defconfig, can also contain config fragments information",
+            "default": "The defconfig value"
+        },
+        "arch": {
+            "type": "string",
+            "description": "The architecture type of this board",
+            "enum": ["arm", "arm64", "x86"],
+            "default": "arm"
+        },
+        "board": {
+            "type": "string",
+            "description": "The name of the board"
+        },
+        "board_instance": {
+            "type": "string",
+            "description": "The instance identifier of the board"
+        },
+        "build_environment": {
+            "type": "string",
+            "description": "Build environment name as used in the build configuration e.g. gcc-8"
+        },
+        "git_branch": {
+            "type": "string",
+            "description": "The branch used for testing"
+        },
+        "metadata": {
+            "type": "object",
+            "description": "Free form object where to store accessory test group data"
+        },
+        "test_cases": {
+            "type": "array",
+            "description": "The list of test case objects executed by this test group",
+            "items": {"$ref": "http://api.kernelci.org/json-schema/1.0/test_case_get.json"},
+            "additionalItems": true
+        },
+        "sub_groups": {
+            "type": "array",
+            "description": "The list of sub-group objects included in this group",
+            "items": {"$ref": "http://api.kernelci.org/json-schema/1.0/test_group_get.json"},
```

Comment:
Shouldn't this be `json-schema/1.2` instead?

Reply:
In general the structure of the json-schema directories is:
```
- schema
  |_ 1.0
     |_ attachment.json
     ...
     |_ get_test_group.json
 |_ 1.1
   |_ ...
 |_ 1.2
   |_...
```

Each version subdirectory contains only the files that were changed in this version of API, therefore test_group_get.json exists only in the version 1.0.
When creating version 1.2 I followed the pattern for version 1.1.

The whole structure looks as if each json is versioned **not** the whole API.

So either should stay as it is or we should change the versioning schema.

Reply:
OK I thought the test group API had changed in version 1.2.  I guess it may all just get removed in the follow-up PR.

Reply:
> OK I thought the test group API had changed in version 1.2. I guess it may all just get removed in the follow-up PR.

I double checked it and you're actually right. This shouldn't point to the GET request JSON at first. It should point to version 1.2 of the POST JSON.